clients/go/apis/v1alpha1/config.go

```diff
@@ -358,3 +349,24 @@ func (r *AuditRule) SetDefault() {
 		r.LogType = AuditLogRequest_DATA_ACCESS.String()
 	}
 }
+
+// Justification specifies the config used to integrate with JVS.
+type Justification struct {
+	// JVSPublicKeysEndpoint is the endpoint where public keys may be retrieved from the JVS.
+	// These will be used to validate JWT tokens that are passed in through the
+	// "justification-token" header.
+	JVSPublicKeysEndpoint string `yaml:"jvs_public_keys_endpoint,omitempty"`
+
+	// Enabled indicates whether enables adding justification information to audit logs or not. If this is enabled,
+	// all manual calls are expected to pass in a justification in the "justification-token" header.
+	// If omitted, justifications will not be added to logs, even if provided.
+	Enabled bool `yaml:"enabled,omitempty"`
+}
+
+// Validate validates the Justification.
+func (justification *Justification) Validate() error {
+	if justification.Enabled && justification.JVSPublicKeysEndpoint == "" {
+		return fmt.Errorf("jvs_public_keys_endpoint must be specified when justification is enabled")
+	}
```

Comment:
nit. I recommend do the following. If we need to add configs in the future.

```suggestion
	if !justification.Enabled {
		return nil
	}
	if justification.JVSPublicKeysEndpoint == "" {
		return fmt.Errorf("jvs_public_keys_endpoint must be specified when justification is enabled")
	}
```

Reply:
I changed it to 
```
	if justification.Enabled {
		if justification.PublicKeysEndpoint == "" {
			return fmt.Errorf("public_keys_endpoint must be specified when justification is enabled")
		}
	}
	return nil
```
I feel this one looks more straightforward